src/test_helpers/mod.rs

```diff
@@ -60,3 +64,28 @@ pub fn enable_enterprise_attestation(
 
     Ok(attestation_material)
 }
+
+pub fn enable_pin_uv(
+    state: &mut CtapState,
+    env: &mut impl Env,
+    pin_uv_auth_protocol: PinUvAuthProtocol,
+    slot_id: usize,
+) -> Result<Vec<u8>, Ctap2StatusCode> {
+    let key_agreement_key = crypto::ecdh::SecKey::gensk(env.rng());
+    let pin_uv_auth_token = [0x91; PIN_TOKEN_LENGTH];
+    let client_pin = ClientPin::new_test(
+        env,
+        slot_id,
+        key_agreement_key,
+        pin_uv_auth_token,
+        pin_uv_auth_protocol,
+    );
+    state.set_client_pin_for_test(client_pin);
+    storage::set_pin(env, slot_id, &[0x88; 16], 4)?;
+
+    Ok(authenticate_pin_uv_auth_token(
+        &pin_uv_auth_token,
+        &DUMMY_CLIENT_DATA_HASH,
```

Comment:
`DUMMY_CLIENT_DATA_HASH` should be passed in, as it seems to be an implementation detail of the caller of `enable_pin_uv`.

Reply:
done